devices/HumanDataCollector/conf/human_data_collector.ini

```diff
@@ -0,0 +1,4 @@
+[plugin human_data_collector]
+type device
+name human_data_collector
+library HumanDataCollector
```

Comment:
Do we still need the `ini` file? I'm not actively using Yarp since few months but I remember that recent versions implemented an automatic way to generate these files. If this is true, we could start with this device, and then update all the others in another PR.

cc @traversaro

Reply:
Exactly, since YARP 3.2.0, see http://www.yarp.it/devel/v3_2_0.html . Note that you need to specify the `YARP_INI DESTINATION` argument to `yarp_install` to correctly install the autogenerated `.ini` file.

Reply:
Thanks @traversaro

Reply:
Ref: https://github.com/robotology/yarp/pull/1964/commits/0606d1cee133d177b1c2c00f315e237988f2d387